wiki/Ranking_Criteria/fr.md

```diff
@@ -0,0 +1,276 @@
+- **Un objet qui est mal snapped parce qu'il passe ou se termine légèrement avant un nouveau [point de timing non hérité](/wiki/Client/Beatmap_editor/Timing#points-de-timing-non-hérités) doit voir son tail snapped dans la nouvelle section de timing.** Pour les spinners et les notes longues d'osu!mania, cela peut être réalisé en faisant glisser l'extrémité de l'objet dans la timeline. Pour les sliders, cela peut être réalisé en manipulant le slider velocity ou en éditant le fichier `.osu`.
+
+## Audio
+
+### Règles
+
+- **Le fichier audio d'une beatmap doit utiliser le format de fichier `.mp3` ou `.ogg` et avoir un débit binaire moyen ne dépassant pas 192kbps.**
+- **Une beatmap ne peut contenir qu'un seul fichier d'une musique utilisé par toutes les difficultés.** Les fichiers de musique multiples dans une seule beatmap ne sont pas pris en charge et entraînent un comportement inattendu avec les temps de prévisualisation, les métadonnées, etc.
+- **Le fichier audio et les fichiers hitsound d'une musique doivent être de qualité raisonnable.** Essayez de trouver le fichier source de la plus haute qualité disponible plutôt que d'extraire un fichier d'un site de vidéo en continu. Les musiques doivent être normalisées en fonction de leur volume de sortie original et ne doivent pas être encodées à un débit binaire supérieur à celui de leurs fichiers originaux.
+- **Les beatmaps doivent avoir des hitsounds.** Les hitnormal donnent un feedback au joueur, et les ajouts (whistles, claps, et finishes) accentuent les parties les plus importantes de la musique.
+  - **Les beatmaps d'osu!mania ne nécessitent pas l'ajout d'hitsound.** Cela permet de faciliter l'approche des mappeurs osu!mania de différentes origines. Il est toujours fortement recommandé d'ajouter des hitsounds afin d'améliorer la sensation de vos beatmaps. Dans les cas où les hitsounds ne sont pas utilisés, *aucun* ajout ne peut être placé, et *aucun* point de contrôle de l'échantillon ne doit être utilisé (par exemple pour ajuster le volume).
+- **Tous les objets sur lesquels on clique doivent avoir des [hitsounds actifs](/wiki/Beatmapping/Hitsound#hitsound-actif) audibles.** Les difficultés d'osu!mania en sont exemptes en raison de la construction rythmique du mode.
+- **Les [hitsounds actifs](/wiki/Beatmapping/Hitsound#hitsound-actif) doivent utiliser les formats de fichier `.wav` ou `.ogg`.** Les fichiers `.mp3` ont de légers retards, et sont donc réservés aux [hitsounds passifs](/wiki/Beatmapping/Hitsound#hitsound-passif), plus longs comme les bruits d'ambiance.
+- **Les [hitsounds actifs](/wiki/Beatmapping/Hitsound#hitsound-actif) ne peuvent avoir un délai supérieur à 5 millisecondes.** "Le "Delay" prend en compte le pic d'un son plutôt que son introduction. Cela permet de synchroniser efficacement le retour du hitsound avec une musique. Le skin par défaut `normal-hitfinish.wav` a un léger retard, mais peut toujours être utilisé comme un hitsound personnalisé.
```

Comment:
Still needs to be updated according to https://github.com/ppy/osu-wiki/pull/6828